server/service/integration_mdm_test.go

```diff
@@ -10590,7 +10578,7 @@ func (s *integrationMDMTestSuite) TestManualEnrollmentCommands() {
 	checkInstallFleetdCommandSent(mdmDevice, false)
 }
 
-func (s *integrationMDMTestSuite) TestCustomConfigurationWebURL() {
+func (s *integrationMDMTestSuite) TestZCustomConfigurationWebURL() {
```

Comment:
Typo `Z`?

Reply:
sorry, I should've added a note. It's so this test runs last, it's setting some state that makes multiple tests fail.

my plan is to get to this and the skipped test tomorrow

Reply:
Gotcha!